enterprise-suite/es-grafana/cassandra-exporter.json

```diff
@@ -2,6 +2,6 @@
   {"graphName":"Storage", "promQL":["irate(cassandra_storage_load{workload=\"$workloadId\"}[1m])"]},
   {"graphName":"Reads", "promQL":["irate(cassandra_clientrequest_latency{clientrequest='Read', workload=\"$workloadId\"}[1m])"]},
   {"graphName":"Writes", "promQL":["irate(cassandra_clientrequest_latency{clientrequest='Write', workload=\"$workloadId\"}[1m])"]},
-  {"graphName":"Read Latency", "promQL":["irate(cassandra_clientrequest_totallatency{clientrequest='Read', workload=\"$workloadId\"}[1m])"]},
-  {"graphName":"Write Latency", "promQL":["irate(cassandra_clientrequest_totallatency{clientrequest='Write', workload=\"$workloadId\"}[1m])"]}
+  {"graphName":"Read Latency", "promQL":["cassandra_clientrequest_totallatency{clientrequest='Read', workload=\"$workloadId\"}"]},
```

Comment:
@srikanthops can you confirm this one?

totallatency sounds like it might be a lifetime counter, in which case we need some kind of derivative, although per/second (rate) is unlikely the right one.

This looks like a good way to graph counters to me:

```
"idelta(cassandra_clientrequest_totallatency{clientrequest='Write', workload=\"$workloadId\"}[1m]) >= 0"
```

But I haven't thought about how to graph a lifetime latency counter.  Same way?  Seems like you'd want to divide by number of requests to get some understandable basis?

Discussion here

https://github.com/lightbend/k8s-explore/pull/129

Reply:
https://gitlab.cncf.ci/prometheus/prometheus/blob/2195bb66f7d04cee8848b6829a710a033cc68e39/consoles/cassandra.html

Reply:
http://cassandra.apache.org/doc/latest/operating/metrics.html

"Latency
Special type that tracks latency (in microseconds) with a Timer plus a Counter that tracks the total latency accrued since starting. The former is useful if you track the change in total latency since the last check. Each metric name of this type will have ‘Latency’ and ‘TotalLatency’ appended to it."

Reply:
I think I'd just revert to the irate way for now.  otherwise you're going to get a monotonic line going up.  we should revisit cassandra latency graph tho, irate makes the y-axis hard to understand